lms/djangoapps/instructor_task/management/commands/fail_old_queueing_tasks.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import unicode_literals, print_function
+
+from datetime import datetime
+
+from celery.states import FAILURE
+from django.core.management.base import BaseCommand, CommandError
+
+from lms.djangoapps.instructor_task.models import InstructorTask, QUEUING
+
+
+class Command(BaseCommand):
+    """
+    Command to manually fail old "QUEUING" tasks in the instructor task table.
+
+    Example:
+    ./manage.py lms fail_old_queueing_tasks --dry-run --after 2001-01-03 \
+        --before 2001-01-06 --task-type bulk_course_email
+    """
+
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
+            '--before',
+            type=str,
+            dest='before',
+            help='Manually fail instructor tasks created before or on this date.',
+        )
+
+        parser.add_argument(
+            '--after',
+            type=str,
+            dest='after',
+            help='Manually fail instructor tasks created after or on this date.',
+        )
+
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help='Return the records this command will update without updating them.',
+        )
+
+        parser.add_argument(
+            '--task-type',
+            dest='task_type',
+            type=str,
+            default=None,
+            help='Specify the type of task that you want to fail.',
+        )
+
+    @staticmethod
+    def parse_date(date_string):
+        """
+        Converts an isoformat string into a python datetime object
+        """
+        return datetime.strptime(date_string, "%Y-%m-%d")
+
+    def handle(self, *args, **options):
+
+        if options['before'] is None:
+            raise CommandError("Must provide a 'before' date")
+
+        if options['after'] is None:
+            raise CommandError("Must provide an 'after' date")
+
+        before = self.parse_date(options['before'])
+        after = self.parse_date(options['after'])
+        filter_kwargs = {
+            "task_state": QUEUING,
+            "created__lte": before,
```

Comment:
Good catch, @jibsheet . Without `utc.localize`, django "converted" them into UTC time. Going to add that to this PR

I inspected the query that the orm generated (once I used utc.localized)—it turns `2017-05-18` into `2017-05-18 00:00:00` (at least for sqlite)